style.css

```diff
@@ -0,0 +1,108 @@
+body {
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  position: static;
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+  color: #6070ff;
+  list-style-type: none;
+}
+
+.navbar_logo a {
+  position: relative;
+  left: 12px;
+  top: 10px;
+  text-decoration: none;
+}
+
+.navbar_logo:hover {
+  width: auto;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.navbar_icon a {
+  position: relative;
+  top: 10px;
+  right: 12px;
+}
+
+.navbar_icon a:hover {
+  width: auto;
+  top: 9px;
+  display: flex;
+  align-items: center;
+}
+
+#headline {
+  background-image: url(./image/background.png);
+  background-size: cover;
+  height: auto;
+  background-repeat: no-repeat;
+  padding-top: 206px;
+  padding-left: 26px;
+  padding-right: 26px;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+}
```

Comment:
- [ ] Please reduce the `padding-top: 206px` it actually generating a lot of space from the `header` to the headline `section`.
### **Take a look**
![padding](https://user-images.githubusercontent.com/55596970/158948931-82f3de91-6e57-43ac-8322-a9caf22b3a22.png)